boards/arm/google_kukui/google_kukui.dts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2019 The Chromium OS Authors
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <st/f0/stm32f091Xc.dtsi>
+
+/ {
+	model = "Google Kukui EC";
+	compatible = "st,stm32f098rc", "st,stm32f098";
+
+	chosen {
+		zephyr,console = &usart1;
+		zephyr,shell-uart = &usart1;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+	};
+};
+
+&usart1 {
+	current-speed = <115200>;
+	pinctrl-0 = <&usart1_pins_a>;
+	pinctrl-names = "default";
```

Comment:
Might need to remove these `pinctrl` definitions for now. See commit 4dc303b99b ("dts: stm32: Remove pinctrl definitions").

DTS `pinctrl` support should be coming soonish. There's alread basic support in edtlib.

Reply:
Indeed, please remove the `pinctrl-xx` for now.

Reply:
Done